src/base/emu-i386/simx86/interp.c

```diff
@@ -2318,14 +2318,11 @@ intop3b:		{ int op = ArOpsFR[D_MO(opc)];
 				unsigned char opm = D_MO(Fetch(PC+2));
 				switch (opm) {
 				case 0: /* SGDT */
-				    /* Store Global Descriptor Table Register */
-				    PC++; PC += ModRM(opc, PC, _mode|DATA16|MSTORE);
-				    error("SGDT not implemented\n");
-				    break;
 				case 1: /* SIDT */
+				    /* Store Global Descriptor Table Register */
 				    /* Store Interrupt Descriptor Table Register */
-				    PC++; PC += ModRM(opc, PC, _mode|DATA16|MSTORE);
-				    error("SIDT not implemented\n");
+				    PC++; PC += ModRM(opc, PC, _mode);
+				    Gen(O_SIM, _mode, 0x101, opm, P0);
```

Comment:
What is 0x101?

Reply:
0x100+opc2, or the extended opcode used in Sim_helper. I've changed it now for consistency with the other cases.